wyvern/feature_store/historical_feature_util.py

```diff
@@ -68,6 +69,15 @@ def build_historical_real_time_feature_requests(
         curr_feature_names,
     ) in features_grouped_by_entity.items():
         entity_list = entity_identifier_type.split(SQL_COLUMN_SEPARATOR)
+
+        if len(entity_list) > 2:
+            return result_dict
+
+        # validate entity_list
+        for entity in entity_list:
+            if entity not in entities:
+                raise EntityColumnMissingError(entity=entity)
+
```

Comment:
We won't run this validation `if len(entity_list) > 2`. Shouldn't we always run the validation?

Reply:
sg! probably eventually it's a noop since entity_identifier_type is probably ensured to be <= 2 at this point. i'll update

Reply:
updated